### ISSUE TITLE
website datatables ajax error + coverage processing not working

### ISSUE THREAD

**Issue by zhorvatic** [lemurheavy/coveralls-public#1619]
The coverage file is successfully posted, returns 200 OK but coverage isn't calculated. Visited the job and got this message:

![image](https://user-images.githubusercontent.com/43344786/155357017-84f4653f-f459-4417-9463-0a7345c1845b.png)

https://coveralls.io/jobs/95057762

Here are additional jobs (for different repos) that don't work properly at this moment:
- https://coveralls.io/jobs/95057931
- https://coveralls.io/jobs/95057864
- https://coveralls.io/jobs/95057864
- https://coveralls.io/jobs/95057776

For one I also couldn't fetch repo token over API (we have it automated as part of CI)
- 10:54:17 23/02/2022
- https://coveralls.io/github/superbet-group/golang.lib

Your status page says everything is functioning ok, which doesn't seem to be the case:
https://status.coveralls.io/ 
![image](https://user-images.githubusercontent.com/43344786/155359946-4830d3d9-aa09-408a-a1d5-e9223bd9ec19.png)

**Comment by afinetooth** [comment]
Hi, @zhorvatic.

**tl;dr**
The "DataTables warning" error indicates a performance slowdown at https://coveralls.io/. The issue itself will eventually resolve and there's nothing you need to do, or can do, to accelerate the process.

**Details**: 
A series of background jobs run behind-the-scenes after you post a coverage job to the Coveralls API that are responsible for completing the build associated with the job (or _jobs_ if you have a parallel build). Those background jobs do things like perform your aggregate coverage calculation and render some interactive portions of your new build page. When https://coveralls.io/ has higher-than-normal traffic, those background jobs can get stuck in queues, waiting to be processed. When they're delayed like that, some of the components on your build page can fail because they're waiting for data that's not yet available at the endpoint they're requesting. (That mainly results in that "DataTables" error.)

While it is an issue that we're trying to address, you can rest assured that, when you see this type of error, your build is in progress and will eventually complete. 

Again, we do our best to address both transient spikes in traffic and traffic trends that dictate infrastructure upgrades. We've already taken some action since last week with permanent infrastructure to address the recent performance issues, and we are working on some solutions to address spikes this week and next, so I hope your builds will complete at least a little more quickly soon. The first upgrades we did pointed to a decrease in build times of around 25% for our test set.

If you continue to see performance like this, though, let us know here, or, ideally, email us at support@coveralls.io with a list of all repos displaying the issue and we'll put you on a list of customers to keep updated as we make further progress.

**Comment by zhorvatic** [comment]
Thank you for the in-depth explanation. I can also see that the builds have been processed now.
